cirq/circuits/circuit.py

```diff
@@ -1387,6 +1389,48 @@ def tetris_concat(
 
         return cirq.Circuit(buffer[offset : offset + n_acc])
 
+    def get_qubit_factors(self) -> List[Set['cirq.Qid']]:
+        """
+        Divide circuit's qubits into qubit factors such that there are
```

Comment:
Try to write docstrings so the first line is only one line long and provides a brief summary. Details can be included in the following paragraph(s). Our style guide says lines can be 100 lines long including docstrings

Reply:
Done.